public/stylesheets/style.css

```diff
@@ -7,11 +11,22 @@
   --color-mid-green: #3f9e44;
   --color-sub-green: #24512c;
   --color-dark-green: #192d24;
-  --color-hero-blue: #61d1fa;
-  --color-mid-blue: #458fab;
-  --color-sub-blue: #284c5d;
-  --color-dark-blue: #1a2a36;
   --color-hero-red: #ff7676; 
+  --color-mid-red: #ae5458;
```

Comment:
Hmm, we should release this as a separate package at some point, would be handy.